src/vfb_connect/cross_server_tools.py

```diff
@@ -114,23 +115,48 @@ def _get_neurons_connected_to(self, neuron, weight, direction, classification=No
                         "downstream.short_form as target_neuron_id, downstream.label as target_neuron_name"
         r = self.nc.commit_list([cypher_query])
         dc = dict_cursor(r)
-        return pd.DataFrame.from_records(dc)
+        if return_dataframe:
+            return pd.DataFrame.from_records(dc)
+        else:
+            return dc
+
+    def get_morphologically_similar_neurons_to(self, neuron, cutoff=None, source=None, return_dataframe=True):
```

Comment:
I know command completion and everything but that's something of a mouthful ... do we need the terminal prepositions? I see it's the style. 

Is this designed to allow for neuronbridge matches also?

Reply:
Happy to lose the terminal preposition in this case but wouldn't shorten much. Naming it for NBLAST would be more compact, but would be nice to keep it broad so we could extend to other algorithms.  BTW are neuronbridge scores available (e.g. via api?)